dev-tools/mage/common.go

```diff
@@ -821,3 +821,28 @@ func ListMatchingEnvVars(prefixes ...string) []string {
 	}
 	return vars
 }
+
+// IntegrationTestEnvVars returns the names of environment variables needed to configure
+// connections to integration test environments.
+func IntegrationTestEnvVars() []string {
+	// Environment variables that can be configured with paths to files
+	// with authentication information.
+	vars := []string{
+		"AWS_SHARED_CREDENTIAL_FILE",
```

Comment:
Do we support both shared credentials and access keys in https://github.com/elastic/beats/pull/17480/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3R924?

Reply:
Not yet, I am leaving the list of variables here for the record.

We would also need to add support for credential files in tests: https://github.com/elastic/beats/blob/a7e588c40af636ff1923dd97b839497a8a901f4b/x-pack/metricbeat/module/aws/mtest/integration.go#L18
And then in jenkins write the file and leave it in some safe place.

I would leave this for a separate change.

Reply:
Sounds good to me! Thank you!